src/box/xrow.c

```diff
@@ -1478,13 +1478,14 @@ xrow_decode_raft(const struct xrow_header *row, struct raft_request *r,
 {
 	if (row->type != IPROTO_RAFT)
 		goto bad_msgpack;
-	if (row->bodycnt != 1 || row->group_id != GROUP_LOCAL) {
+	if (row->bodycnt != 1) {
```

Comment:
After this change, a replica running an older binary won't be able to subscribe to a master running a patched binary. This would break even patch-level upgrades.

Reply:
Resolving since the issue is addressed.